src/utils/graph_building.jl

```diff
@@ -103,6 +116,9 @@ function weights_cutoff(is, js, dists; max_num_nbr = 12, dist_decay_func = inver
 
     # average across diagonal, just in case
     weight_mat = 0.5 .* (weight_mat .+ weight_mat')
+
+    # normalize weights
+    weight_mat = weight_mat ./ maximum(weight_mat)
```

Comment:
so, the weights are normalized by default now?

Reply:
yes, I realized I couldn't think of a reason why you wouldn't want to normalize them (see commit message f5dc410620e39b6c8bb661ce82c9f13089dd54e1 for some other changes)

Reply:
sounds good to me.